init.json

```diff
@@ -0,0 +1,15 @@
+{
```

Comment:
This seems to be here by accident?

Reply:
Oops, had a duplicate clone of the repo and accidentally pushed from it. Should be better now. @lucywyman